features/old/applications/providers/create_application.feature

```diff
@@ -18,7 +18,7 @@ Feature: Create application
 
 
   #TODO move single_app mode examples to separate feature?
-  @javascript
+  @javascript @selenium
```

Comment:
Why it needs selenium now?

Reply:
This was failing on jenkins and passing locally with webkit, maybe now it's good to try it without `@selenium` since capybara it's been updated and circile ci is rocking it